Automatic Validation ended with:
> 2024-02-23T16:44:54.1031161Z ##[error] One or more errors occurred. (Manifest is invalid.) Manifest is invalid. (Line: 22, Col: 5, Idx: 749) - (Line: 22, Col: 5, Idx: 749): While parsing a block mapping, did not find expected key.

(Automated response - build 808.)

--- a/manifests/m/Microsoft/PowerShell/Preview/7.5.0.2/Microsoft.PowerShell.Preview.installer.yaml
+++ b/manifests/m/Microsoft/PowerShell/Preview/7.5.0.2/Microsoft.PowerShell.Preview.installer.yaml
@@ -19,7 +19,7 @@ Installers:
   InstallerUrl: https://github.com/PowerShell/PowerShell/releases/download/v7.5.0-preview.2/PowerShell-7.5.0-preview.2-win-x64.msi
   InstallerSha256: A107F34D9D6E41259D13C891CD2C4062BD5C0FB103A00AD1E525A503B9060897
   ProductCode: '{661AF5A8-2FC6-4BA4-A457-90FE7F8E31CC}'
-    AppsAndFeaturesEntries:
+  AppsAndFeaturesEntries:
   - Publisher: Microsoft Corporation
     DisplayName: PowerShell 7-preview-x64
   ElevationRequirement: elevatesSelf

